code/game/objects/structures/dropship_equipment.dm

```diff
@@ -84,14 +88,14 @@
 
 /obj/effect/attach_point/crew_weapon/dropship1/Initialize()
 	. = ..()
-	ship_tag = "[CONFIG_GET(string/ship_name)] Dropship 1"
+	ship_tag = "alamo"
```

Comment:
Make these at the base typepath since you're no longer calling anything required after compile time